tests/ssg_test_suite/common.py

```diff
@@ -242,42 +245,56 @@ def create_tarball():
         return fp.name
 
 
+def execute_remote_command(machine, args, log_file, error_msg=""):
```

Comment:
There are already similar functions to this, eg. `run_cmd_remote` and `_run_cmd`. I haven't checked, but what do you think? Can we somehow unify them?

Reply:
I would love to, but preferably not in this PR.

Reply:
OK, fair enough